### PR TITLE
Math Magicians: Feature/add UI

### PR DESCRIPTION
**Pull request by mateo951**
- Removed unnecessary files and code from the project
- Added react bug-free code 
- Added calculator UI
- Modified readme file

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Dipeshtwis) [inline src/App.js]: - [ ] The `App.js` component should be class-based. In this milestone, the objective is to practice the class-based React component.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calcContainer">
+        <input type="text" id="result" value="...To be continued" disabled />
```
Comment (Dipeshtwis) [inline src/components/Calculator.js]: - [ ] I would recommend putting `0` by default on the display screen to maintain the standard of the calculator. :stuck_out_tongue:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mateo951, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Well done with deleting all the boilerplate created by the CRA.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (mateo951) [reply src/components/Calculator.js]: Hahaha yeah it was just a silly message for you 😁 Done ✔

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @mateo951 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.
 
 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**